Description: Match marker colorants which use non standard string
Author: Peter Hurley <peter@hurleysoftware.com>
Bug: http://www.cups.org/str.php?L4019
Ubuntu-Bug: https://bugs.launchpad.net/ubuntu/+source/cups/+bug/925086

--- a/backend/snmp-supplies.c
+++ b/backend/snmp-supplies.c
@@ -790,7 +790,7 @@
       if (supplies[j].colorant == i)
       {
 	for (k = 0; k < (int)(sizeof(colors) / sizeof(colors[0])); k ++)
-	  if (!strcmp(colors[k][0], (char *)packet->object_value.string.bytes))
+	  if (!strcasecmp(colors[k][0], (char *)packet->object_value.string.bytes))
 	  {
 	    strcpy(supplies[j].color, colors[k][1]);
 	    break;
